pypaas/checkout.py

```diff
@@ -58,7 +58,8 @@ def cmd_env(self):
         env = dict()
         env.update(os.environ)
         if 'env' in self.branch.config:
-            env.update(self.branch.config['env'])
+            for k, v in self.branch.config['env'].items():
+                env[k] = os.path.expandvars(v)
```

Comment:
Das hat zur Folge, dass nur Umgebungsvariablen von pypaas genutzt werden können - nicht weitere zuvor definierte.

Reply:
Wie meinst du mit weiteren variablen? Welche die in davor in dem env block sind? Oder wie?